baseapp/grpcrouter.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 type QueryRouter interface {
-	HybridHandlerByRequestName(name string) []func(ctx context.Context, req, resp protoiface.MessageV1) error
+	HandlerByRequestName(name string) []func(ctx context.Context, req, resp protoiface.MessageV1) error
```

Comment:
Do we need to break this interface name?

This is a public interface and could break consumers

Reply:
I think it is fine, sourcegraph tells me there are no users.
We ask user to migrate to use core services anyway via environment, so those API should be hidden from them.

Reply:
Sure, but couldn't we lower the impact of this change by just maintaining the name of this function for the interface?

Reply:
Do we gain anything by breaking this name?

Reply:
The impact is low as there are no consumers. Hybrid handlers never got fully used as we decided after 0.50 to not support proto v1 & proto v2. Keeping such API name is more confusing than simplifying the API imho.

Reply:
since there are no consumers and the code is no longer working hybrid i feel like its fine. Currently users dont make this far and the ones that do cant actually swap the router other than in 52+ because there was no interface before and the baseapp struct took the concrete implementation.